<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>542</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1008</width>
    <height>542</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UAV</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myFiles.qrc">
    <normaloff>:/new/prefix1/uav.png</normaloff>:/new/prefix1/uav.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QFrame" name="frame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="8" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>FPS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser">
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>20</height>
              </size>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>362</width>
               <height>329</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>Source</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>361</width>
               <height>329</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>Target</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Source Image Stream</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Output Image Stream</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>86</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Output</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Tracked Buoys</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Canny</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Thresh_Red</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Thresh_Green</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Thresh_Yellow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>R</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>G</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Thresh_Flat</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>Filters</string>
             </attribute>
             <widget class="QWidget" name="widget" native="true">
              <property name="geometry">
               <rect>
                <x>-10</x>
                <y>0</y>
                <width>171</width>
                <height>381</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>171</width>
                <height>381</height>
               </size>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>148</width>
                 <height>340</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="5" column="0">
                 <widget class="QCheckBox" name="checkBox_4">
                  <property name="text">
                   <string>Split RGB</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QCheckBox" name="checkBox_9">
                  <property name="text">
                   <string>Thresh Red</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QGroupBox" name="canny_group">
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="title">
                   <string>Canny Filters</string>
                  </property>
                  <property name="flat">
                   <bool>false</bool>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <widget class="QRadioButton" name="radioButton">
                     <property name="text">
                      <string>canny_red</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioButton_2">
                     <property name="text">
                      <string>canny_green</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioButton_3">
                     <property name="text">
                      <string>canny_blue</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioButton_5">
                     <property name="text">
                      <string>canny_binary</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QCheckBox" name="checkBox_8">
                  <property name="text">
                   <string>Thresh Green</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QCheckBox" name="checkBox_7">
                  <property name="text">
                   <string>Thresh Yellow</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QCheckBox" name="checkBox_5">
                  <property name="text">
                   <string>CHough</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QCheckBox" name="checkBox">
                  <property name="text">
                   <string>Canny</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QCheckBox" name="checkBox_2">
                  <property name="text">
                   <string>Disable Cam</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </widget>
            <widget class="QWidget" name="tab_6">
             <attribute name="title">
              <string>Settings</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>&lt;b&gt;HSV&lt;/b&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QRadioButton" name="YellowHSV">
                    <property name="text">
                     <string>Yellow</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="GreenHSV">
                    <property name="text">
                     <string>Green</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="RedHSV">
                    <property name="text">
                     <string>Red</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QWidget" name="UpperHSV" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QSpinBox" name="spinBox"/>
                     </item>
                     <item>
                      <widget class="QSlider" name="horizontalSlider">
                       <property name="maximumSize">
                        <size>
                         <width>91</width>
                         <height>29</height>
                        </size>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QSpinBox" name="spinBox_2"/>
                     </item>
                     <item>
                      <widget class="QSlider" name="horizontalSlider_2">
                       <property name="maximumSize">
                        <size>
                         <width>91</width>
                         <height>29</height>
                        </size>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <item>
                      <widget class="QSpinBox" name="spinBox_3"/>
                     </item>
                     <item>
                      <widget class="QSlider" name="horizontalSlider_3">
                       <property name="maximumSize">
                        <size>
                         <width>91</width>
                         <height>29</height>
                        </size>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="LowerHSV" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_8">
                     <item>
                      <widget class="QSpinBox" name="spinBox_4"/>
                     </item>
                     <item>
                      <widget class="QSlider" name="horizontalSlider_4">
                       <property name="maximumSize">
                        <size>
                         <width>91</width>
                         <height>29</height>
                        </size>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_9">
                     <item>
                      <widget class="QSpinBox" name="spinBox_5"/>
                     </item>
                     <item>
                      <widget class="QSlider" name="horizontalSlider_5">
                       <property name="maximumSize">
                        <size>
                         <width>91</width>
                         <height>29</height>
                        </size>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_10">
                     <item>
                      <widget class="QSpinBox" name="spinBox_6"/>
                     </item>
                     <item>
                      <widget class="QSlider" name="horizontalSlider_6">
                       <property name="maximumSize">
                        <size>
                         <width>91</width>
                         <height>29</height>
                        </size>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Page</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Video_Feed"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSave"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Video_Feed">
   <property name="text">
    <string>Open Video Feed</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="myFiles.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>888</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>840</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>815</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>887</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>886</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>842</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>830</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>894</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>880</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>818</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>816</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>895</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>824</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>897</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>826</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>890</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>814</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>822</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>920</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>927</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>809</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>928</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
